src/components/ManageAccountsDrawer/components/ImportAccounts.tsx

```diff
@@ -166,11 +168,14 @@ export const ImportAccounts = ({ chainId, onClose }: ImportAccountsProps) => {
     () => wallet instanceof MetaMaskShapeShiftMultiChainHDWallet,
     [wallet],
   )
+  const isMetaMaskButNotSnap = isMetaMaskMultichainWallet && isSnapInstalled === null
```

Comment:
this is more like `isMetaMaskButWaitingForSnapInstallCheck` which is not an easy one to name, feels better inline imo

Reply:
Yeah let's inline it for consistency and to keep things clean, also we want to check for `!isSnapInstalled` instead of null (`useIsSnapInstalled()` returns `boolean | null` i.e null while loading, a boolean then

Reply:
we want to keep it null, it is basically making sure snap state is initialized before enabling the account metadata query if it is metamask wallet

Reply:
Well, I forgot to remove this actually, my bad

Reply:
@kaladinlight sounds like `!isSnapInstalled` would catch it regardless? i.e 
`isMetaMaskMultichainWallet && !isSnapInstalled` means the query won't be enabled until `isSnapInstalled` is true (and won't be if we're loading, and also won't if snap isn't installed).

If we do need the loading granularity in the one spot (loading more) but don't need it in the other (initial account meta fetching) then we should use two expressions, one for loading state, the other for *actual* snap installed checks

Reply:
multi account is guarded in the `getAccountIdsWithActivityAndMetadata` so my original thought was that we should trigger query regardless, but it doesn't benefit us in any way. agree with your call, no need for explicit null

Reply:
Yeah we would no-op there either way but that would be a dead click haha https://jam.dev/c/16b5c06a-77fa-4ff5-9e02-0402273e4673